### ISSUE TITLE
CosmWasm Query msgs should be encoded with UTF-8

### ISSUE THREAD

**Issue by ud-klee** [cosmos/cosmjs#1269]
Currently it is using ASCII https://github.com/cosmos/cosmjs/blob/v0.29.0-rc.1/packages/cosmwasm-stargate/src/modules/wasm/queries.ts#L120

There was a PR for `ExecuteMsg` https://github.com/cosmos/cosmjs/issues/697, shouldn't it be applied to queries as well? I just tried to query a contract with Unicode contents and got

```
Error: Cannot encode character that is out of printable ASCII range: 38283
    ...
    at Object.queryContractSmart (/home/scott/tiger/node_modules/@cosmjs/cosmwasm-stargate/src/modules/wasm/queries.ts:120:63)
    at CosmWasmClient.queryContractSmart (/home/scott/tiger/node_modules/@cosmjs/cosmwasm-stargate/src/cosmwasmclient.ts:443:52)
```

The same query can be executed using the Cosmos CLI with `q wasm state smart --b64` and Base64 encoded payload.

**Comment by webmaster128** [comment]
That's a bug. It must be `toUtf8`. Thank you!

**Comment by webmaster128** [comment]
Shipped as part of CosmJS 0.29.0
